lib/iris/experimental/um.py

```diff
@@ -726,20 +725,43 @@ def cutout(ffv_src, filename, cutout_params):
 
     * ffv_src (:class:`FieldsFileVariant`)
         A source FieldFileVariant to subset.
-    * filename (string)
-        The name of the file containing the subset of the source UM
-        FieldsFile variant.
+    * filepath (string)
```

Comment:
I'm not saying 'filename' is wrong, I just find 'filepath' clearer.
I try to always make this distinction as it's been so helpful when messing with paths, especially in our hopefully OS-neutral context.  So I think it has become automatic.
Actually, I should edit the description as well...